jinahub/encoders/audio/VGGISHAudioEncoder/vggish_audio_encoder.py

```diff
@@ -17,30 +22,63 @@ class VggishAudioEncoder(Executor):
     """
      Encode audio data with Vggish embeddings
 
-     :param model_path: path of the pre-trained Vggish model
-     :param pca_path: path of the pre-trained PCA model
+     :param model_path: path of the models directory
      :param default_traversal_paths: fallback batch size in case there is not batch size sent in the request
      """
 
     def __init__(self,
-                 model_path: str = os.path.join(cur_dir, 'models/vggish_model.ckpt'),
-                 pca_path: str = os.path.join(cur_dir, 'models/vggish_pca_params.npz'),
+                 model_path: str = os.path.join(cur_dir, 'models'),
```

Comment:
```suggestion
                 model_path: str = cur_dir / 'models'
```

Reply:
Addressed, please see [here](https://github.com/jina-ai/executors/pull/31/commits/d5752827dc4e83eb858cb90d75d24d2b01830e0e) for the change, thank you.